track-modules.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+deps(){
+test -n "$1" || return
+	n=$1
+	echo $n
+/usr/sbin/modinfo -k 5.4.48-gentoo-tormenta16 --basedir sample --field depends $n | tr "," "\n" | \
```

Comment:
Identation is misaligned (here and below)

Reply:
Fixed